Content.Server/GameTicking/Rules/AllCaptainsRuleSystem.cs

```diff
@@ -0,0 +1,73 @@
+using System.Linq;
+using Content.Server.GameTicking.Presets;
+using Content.Server.GameTicking.Rules.Configurations;
+using Content.Server.Station.Components;
+using Content.Shared.Random;
+using Content.Shared.Random.Helpers;
+using Robust.Shared.Map;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Configuration;
+using Content.Shared.CCVar;
+
+namespace Content.Server.GameTicking.Rules;
+
+public sealed class AllCaptainsRuleSystem : GameRuleSystem
+{
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly GameTicker _ticker = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+    public override string Prototype => "AllCaptains";
+
+    private EntityUid holdJobs; // dunno if there should only be one reference like this because I'm a ss14 noob but hey  it's only gotta work one day :P
+
+    public override void Added()
+    {
+    }
+
+    public override void Started()
+    {
+        // temporarily disable role timers -- super hacky way workaround for client to be aware that role timers aren't required
+        // without having to set up some kind of replication
+        _cfg.SetCVar(CCVars.GameRoleTimers, false);
+    }
+
+    public override void Ended()
+    {
+        _cfg.SetCVar(CCVars.GameRoleTimers, true);
+    }
+
+    public StationJobsComponent GetJobs(EntityUid station)
+    {
+        if (!holdJobs.IsValid() || !HasComp<StationJobsComponent>(holdJobs)) // this doesn't check station parameter since all captains mode is the same for all stations.
```

Comment:
how the fuck does this not explode violently

Reply:
@Skyedra oh
im sorry!
I'm just surprised. I can show you how to do this properly in a bit (currently a tad sick)
I did not mean that in any harsh way directed at you.

Reply:
No worries and thanks again for the detailed info over Discord, really helped me learn more about the codebase :+1: